src/org/usfirst/frc/team1024/robot/commands/PushGearCommand.java

```diff
@@ -0,0 +1,29 @@
+package org.usfirst.frc.team1024.robot.commands;
+
+import org.usfirst.frc.team1024.robot.subsystems.Gear;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class PushGearCommand extends Command {
+	Gear gear;
+	
+	public PushGearCommand(Gear gear) {
+		super("PushGearCommand");
+		requires(gear);
+	}
+	
+	@Override
+	protected boolean isFinished() {
+		
+		// TODO Auto-generated method stub
+		
+		return false;
+		
+	}
+	
+	@Override
+	protected void execute() {
+		// TODO Execute the "PushGear" code
```

Comment:
Do we also need to release the side clamps on the gear before we push the gear out?